boilerplate/openshift/osd-container-image/standard.mk

```diff
@@ -76,10 +80,12 @@ isclean:
 .PHONY: osd-container-image-build
 osd-container-image-build: isclean
 	${CONTAINER_ENGINE} build --pull -f $(DOCKERFILE) -t $(IMAGE_URI) .
+	${CONTAINER_ENGINE} tag $(IMAGE_URI) $(IMAGE_URI_LATEST)
```

Comment:
we can add an additional -t flag to build process like the following:
```shell
${CONTAINER_ENGINE} build --pull -f $(DOCKERFILE) -t $(IMAGE_URI) -t $(IMAGE_URI_LATEST) .
```
so no need to call the bin second time.

Reply:
Neat! Implemented in latest commit. Thanks for the feedback.